src/Build/Graph/ProjectGraph.cs

```diff
@@ -12,6 +14,9 @@ namespace Microsoft.Build.Graph
     /// </summary>
     public sealed class ProjectGraph
     {
+        private const string ProjectReferenceTargetsItemType = "ProjectReferenceTargets";
+        private const string ProjectReferenceTargetsMetadataName = "Targets";
```

Comment:
We've been putting special element names in MSBuildConstants

Reply:
Sure I can do that if that's the pattern, but it's a bit of a weird pattern that a constant used in literally only 1 class would be in some central location. "internal const" I get, but "private const" seems fine to be like this.